src/Uno.Extensions.Navigation.Toolkit/Uno.Extensions.Navigation.Toolkit.UI.csproj

```diff
@@ -19,7 +19,22 @@
 		<PackageReference Include="Uno.UI" Version="4.3.6" />
 		<PackageReference Include="Uno.Toolkit.UI" Version="2.0.0" />
 	</ItemGroup>
-	
+
 	<Import Project="common.props" />
 
+	<!--
+	Fix for: https://github.com/unoplatform/uno.extensions/issues/563 
+	MSBuild Extras excludes .xf.xml layout files from referenced libraries at
+	https://github.com/novotnyllc/MSBuildSdkExtras/blob/252ba9af3300db98fde54644316567f0c6426e1e/Source/MSBuild.Sdk.Extras/Build/Workarounds.targets#L94
```

Comment:
Perhaps, or at least add a discussion about it. I'm not sure this is necessarily a bug that should be fixed in the general case as it may result in errorneous files being added to the package. What's weird is that the Navigation.UI.xr.xml needs to be in the package at all - this to me seems to be the error